addons/binding/org.openhab.binding.loxone/README.md

```diff
@@ -0,0 +1,125 @@
+    * _User authentication timeout_
+        * Authentication procedure took too long time and Miniserver closed connection. It should not occur under normal conditions and may indicate performance issue on binding's OS side.
+    * _Timeout due to no activity_
+        * Miniserver closed connection because there was no activity from binding. It should not occur under normal conditions, as it is prevented by sending keep-alive messages from the binding to the Miniserver. By default Miniserver's timeout is 5 minutes and period between binding's keep-alive messages is 4 minutes. If you see this error, try changing the keep-alive period in binding's configuration to a smaller value.
+    * _Other_
+        * An exception occured and its details will be displayed
+
+## Advanced Parameters
+
+### Timeouts
+
+Timeout values control various parts of Websocket connection management. They can be tuned, when abnormal behavior of the binding is observed, which can be attributed to timing.
+
+Timeout values can be changed in advanced parameters section of the thing's configuration page.
+
+* _First connection delay_
+    * Time in seconds between binding initialization with all necessary parameters and first connection attempt.
+    * Range: 0-120 s, default: 1 s
+* _Period between connection keep-alive messages_
+    * Time in seconds between sending two consecutive keep-alive messages, in order to inform Miniserver about active connection and prevent it from disconnecting.
+    * Range: 1-600 s, default: 240 s (4 minutes, Miniserver default connection timeout is 5 minutes)
+* _Connect error delay_
+    * Time in seconds between failed Websocket connect attempt and another attempt to connect. Websocket connection is established before authentication and data transfer. It can usually fail due to unreachable Miniserver.
+    * Range: 0-600 s, default: 10 s
+* _Response timeout_
+    * Time to wait for a response from Miniserver to a request sent from the binding. A request can be any of: websocket connect request, credentials hashing key request, configuration request, enabling of state updates (until initial states are received). If this time passed without the expecte reaction from the Miniserver, connection will be closed. A new connection attempt may be made, depending on the situation.
+    * Range: 0-60 s, default: 4 s
+* _Authentication error delay_
+    * Time in seconds between user authentication error and another connection attempt. User authentication error can be a result of a wrong name or password, or no authority granted to the user on the Miniserver. If this time is too short, Miniserver will eventually lock out user for longer period of time due to too many failed login attempts. This time should allow administrator to fix authentication issue without being locked out. Connection retry is required, because very rarely Miniserver seems to reject correct credentials, which pass on another exactly same attempt.
```

Comment:
"lock out the user for a longer period"

"should allow the administrator to fix the authentication"

"which pass...." -> "which are successful on a subsequent identical attempt"